dask_image/imread/__init__.py

```diff
@@ -43,6 +45,12 @@ def imread(fname, nframes=1):
     if (nframes != -1) and not (nframes > 0):
         raise ValueError("`nframes` must be greater than zero.")
 
+    if arraytype == "numpy":
+        arrayfunc = numpy.asanyarray
+    elif arraytype == "cupy":
+        import cupy
+        arrayfunc = cupy.asanyarray
```

Comment:
I'm sure there's a nicer way to handle this instead of an if clause, this could be refactored (now or later) if someone has a better idea for it.